src/ontology/uberon-edit.obo

```diff
@@ -224708,6 +224685,47 @@ relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0007365 ! rumen
 property_value: dcterms-date 2023-02-08T08:53:48Z xsd:dateTime
 
+[Term]
+id: UBERON:8600000
+name: lobular bronchiole
+def: "A conductive airway that branches into terminal bronchioles." [PMID:16543587, PMID:27611890]
```

Comment:
To note, PMID:16543587 is behind a paywall. PMID:27611890 mentions "preterminal bronchiole", but not "lobular bronchiole".